src/index.html

```diff
@@ -9,6 +12,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <img class="img" src="images/logo.png"
+      alt="logo">
+      <nav class="nav">
+        <a class="link is-active" href="#apple">APPLE</a>
```

Comment:
for better UX you should put the links in a list

Reply:
and don't use upper case for text
for this use CSS rule

Reply:
follow naming convention `block__element--modifier`
in your case link should have class `nav__link`